helper/inheritance-resolver.js

```diff
@@ -11,8 +11,17 @@ module.exports = function(plugins, config, name) { // eslint-disable-line func-n
   }
 
   function generateSymlinks(src, dest, replacePattern, ignore = []) {
+    var themeIgnoreFiles = [];
+    // Iterate through the config.themes to grab any ignored paths that
+    // themers want to be excluded from the gulp.src() and sass compilation
+    Object.keys(config.themes).forEach(name => {
```

Comment:
Understandable but that was the one choke point that I could identify. Open to suggestions on how to allow directories/files to be excluded as I think that has to be considered. I originally thought the helper/scss would be good chokepoint but wasn't having any luck on getting the files to be ignored from the `gulp.src()`